packages/okta-vue/README.md

```diff
@@ -50,7 +52,8 @@ Vue.use(Auth, {
   issuer: 'https://{yourOktaDomain}.com/oauth2/default',
   clientId: '{clientId}',
   redirectUri: 'http://localhost:{port}/implicit/callback',
-  scopes: ['openid', 'profile', 'email']
+  scopes: ['openid', 'profile', 'email'],
+  pkce: true
```

Comment:
Why only show the config change in vue?

@aarongranick-okta PKCE is your jam, does this PR cover everything?